moulinette/authenticators/__init__.py

```diff
@@ -135,33 +144,62 @@ def _open_sessionfile(self, session_id, mode='r'):
         return open_cachefile('%s.asc' % session_id, mode,
                               subdir='session/%s' % self.name)
 
-    def _store_session(self, session_id, session_hash, password):
-        """Store a session and its associated password"""
-        gpg = gnupg.GPG()
-        gpg.encoding = 'utf-8'
-
-        # Encrypt the password using the session hash
-        s = str(gpg.encrypt(password, None, symmetric=True, passphrase=session_hash))
-        assert len(s), "For some reason GPG can't perform encryption, maybe check /root/.gnupg/gpg.conf or re-run with gpg = gnupg.GPG(verbose=True) ?"
+    def _store_session(self, session_id, session_token):
+        """Store a session to be able to use it later to reauthenticate"""
 
+        # We store a hash of the session_id and the session_token (the token is assumed to be secret)
+        to_hash = "{id}:{token}".format(id=session_id, token=session_token)
+        hash_ = hashlib.sha256(to_hash).hexdigest()
         with self._open_sessionfile(session_id, 'w') as f:
-            f.write(s)
+            f.write(hash_)
 
-    def _retrieve_session(self, session_id, session_hash):
-        """Retrieve a session and return its associated password"""
+    def _authenticate_session(self, session_id, session_token):
+         """Checks session and token against the stored session token"""
```

Comment:
```suggestion
        """Checks session and token against the stored session token"""
```
This extra space breaks the tests and the linting